sql/codegen.go

```diff
@@ -290,7 +309,7 @@ eval_result = classifier.evaluate(eval_input_fn(X, Y, BATCHSIZE), verbose=0)
 print("Training set accuracy: {accuracy:0.5f}".format(**{"accuracy": eval_result[1]}))
 {{else}}
 eval_result = classifier.evaluate(
-	input_fn=lambda:eval_input_fn(X, Y, BATCHSIZE), steps=STEP)
+	input_fn=lambda:eval_input_fn(X, Y, BATCHSIZE), steps=STEPS)
```

Comment:
What if `STEPS` is larger than the eval dataset batch numbers?

I don't think `steps` should be configurable. A `PREDICT` should always go through the whole selected data. If a user wants a subset of the select, (s)he can put the restriction on the StandardSelect
```sql
SELECT ...
WHERE ...
LIMIT ...
PREDICT...
```

Reply:
Determine the actual dataset size ( total number of elements ) sometimes is time-consuming especially on very large datasets ( like dataset stored in Hive ), if user didn't provide `LIMIT` clause then we must execute `SELECT count(*)  FROM table WHERE ...` then calculate `STEPS`.

I'm not sure whether we should do this now, so put `STEPS` configurable for now and move on to an automatic solution.

Reply:
We don't need to determine the actual dataset size. If we don't specify `steps`, `classifier.evaluate` will consume all the data points returned by`eval_input_fn`.

Reply:
Checked keras's document: https://keras.io/models/model/#fit `model.fit` can accept `steps_per_epoch=None` also. Removing this in the next PR while I'll refine the data reading part